src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -46,6 +43,14 @@ export class IeObjectsController {
 		// Filter on format video should also include film format
 		checkAndFixFormatFilter(queryDto);
 
+		// Get sector from Organisation when user is part of CP_ADMIN Group
+		let organisation = null;
+		if (user.getGroupId() === Group.CP_ADMIN) {
+			organisation = await this.organisationService.findOrganisationBySchemaIdentifier(
+				user.getMaintainerId()
+			);
+		}
```

Comment:
It might be useful to also select sector when the user is first logged in, so the sector is always available on the session user object.

Reply:
https://github.com/viaacode/hetarchief-proxy/blob/master/src/modules/auth/controllers/het-archief.controller.ts#L128-L130